wishlist/wishlist.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Wishlist</title>
+        <link rel="stylesheet" href="/style.css" />
+        <link rel="stylesheet" href="/wishlist/wishlist.css" />
+        <link rel="stylesheet" href="/productpage/product.css" />
```

Comment:
why are you using product & wishlist css if it's already included in style css ?